soundio/soundio.h

```diff
@@ -179,6 +183,8 @@ enum SoundIoChannelId {
     SoundIoChannelIdAux13,
     SoundIoChannelIdAux14,
     SoundIoChannelIdAux15,
+
+    SoundIoChannelIdMax
```

Comment:
we can get rid of this as well as SoundIoChannelLayoutIdMax if we do the other suggestion of just having the static initializer array line up with the enum sans the fancy [x] = {} syntax.